http/jaxrs-reactive/src/test/java/io/quarkus/ts/jaxrs/reactive/HttpCachingResourceIT.java

```diff
@@ -38,6 +39,7 @@ public void shouldGetCacheControlAllAttributes() {
                                 containsString("private"))));
     }
 
+    @DisabledOnQuarkusVersion(version = "2.2.1.*", reason = "https://github.com/quarkusio/quarkus/issues/19822")
```

Comment:
Do you think a `//TODO` for removal could be useful here? I know that the version parameter makes sure it will run on updated version, but it's still an extra line :)